src/mmw/apps/bigcz/serializers.py

```diff
@@ -3,9 +3,18 @@
 from __future__ import unicode_literals
 from __future__ import division
 
+from math import ceil
+
 from rest_framework.serializers import \
     Serializer, CharField, DateTimeField, IntegerField, SerializerMethodField
 from rest_framework_gis.serializers import GeometryField
+from rest_framework.utils.urls import remove_query_param, replace_query_param
+
+from django.conf import settings
+
+# Cannot import CATALOG_NAME as CUAHSI from apps.bigcz.clients.cuahsi.search
+# because that creates circular dependencies
+CUAHSI = 'cuahsi'
```

Comment:
We could consider adding an attribute to each client like`is_pageable`, and adding it to the serializer's context. Then  the checks in `get_page()`, etc. wouldn't have to be specific on CUAHSI, but on `self.context.get('is_pageable')`